specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/connectors.json

```diff
@@ -0,0 +1,618 @@
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Connector setting data",
+          "$ref": "#/definitions/ConnectorSettingProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ConnectorSettingProperties": {
+      "type": "object",
+      "description": "Describes properties of an connector setting",
+      "properties": {
+        "hybridComputeSettings": {
+          "description": "Settings for hybrid compute management, these settings are relevant only Arc autoProvision (Hybrid Compute).",
+          "type": "object",
+          "$ref": "#/definitions/HybridComputeSettingsProperties"
+        },
+        "authenticationDetails": {
+          "description": "Settings for authentication management, these settings are relevant only for the cloud connector.",
+          "type": "object",
+          "$ref": "#/definitions/AuthenticationDetailsProperties"
+        }
+      }
+    },
+    "HybridComputeSettingsProperties": {
+      "type": "object",
+      "description": "Settings for hybrid compute management",
+      "properties": {
+        "hybridComputeProvisioningState": {
+          "description": "State of the service principal and its secret",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Valid",
+            "Invalid",
+            "Expired"
+          ],
+          "x-ms-enum": {
+            "name": "hybridComputeProvisioningState",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Valid",
+                "description": "Valid service principal details."
+              },
+              {
+                "value": "Invalid",
+                "description": "Invalid service principal details."
+              },
+              {
+                "value": "Expired",
+                "description": "the service principal details are expired"
+              }
+            ]
+          }
+        },
+        "autoProvision": {
+          "type": "string",
+          "description": "Whether or not to automatically install Azure Arc (hybrid compute) agents on machines",
+          "enum": [
+            "On",
+            "Off"
+          ],
+          "x-ms-enum": {
+            "name": "autoProvision",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "On",
+                "description": "Install missing Azure Arc agent on VMs automatically"
+              },
+              {
+                "value": "Off",
+                "description": "Do not install Azure Arc agent on the VMs automatically"
+              }
+            ]
+          }
+        },
+        "resourceGroupName": {
+          "type": "string",
+          "description": "The name of the resource group."
+        },
+        "region": {
```

Comment:
in this scenario, when customer is creating a connector to other clouds, what does region or resource group imply? which region is this?

Reply:
As I wrote before regarding Azure Security Center scenario, we connect non-Azure Cloud VMs to Azure using Arc, these parameters are needed to onboard Arc agents,
- resource groups is used to connect the Arc to
- region is indication to the VM region